doc/nrf/app_dev/tfm/index.rst

```diff
@@ -129,25 +129,17 @@ There are several options to get UART output from the secure TF-M:
 Limitations
 ***********
 
-The following services are not supported:
+The following limitations apply to TF-M and its usage:
 
-* Audit Log service in IPC model
-* Firmware Update service
-* Attestation service
+* Firmware Update service is not supported.
+* The following crypto modules or ciphers are not supported:
 
-The following Crypto modules or ciphers are not supported:
+  * Output feedback (OFB) mode.
+  * Cipher feedback (CFB) mode.
 
-* OFB
-* CFB
-
-Isolation level 3 is not supported.
-
-In Isolation level 2 or higher, the number of peripherals configured as secure in Application Root of Trust (ARoT) is limited by the number of available MPU regions.
-
-Nordic platforms support only the GCC toolchain for building TF-M.
-
-Enabling Floating point support in TF-M is currently not supported.
-Enabling Floating point Hard ABI (:kconfig:option:`CONFIG_FP_HARDABI`) in the application is currently not supported.
+* Isolation level 3 is not supported.
+* In Isolation level 2 or higher, the number of peripherals configured as secure in Application Root of Trust (ARoT) is limited by the number of available MPU regions.
+* Nordic platforms support only the GCC toolchain for building TF-M.
```

Comment:
Not sure about the Nordic platforms here... Nordic Semiconductor platforms? Or is nRF Connect SDK platforms more accurate?

Reply:
Whatever you want to say to refer to our SoCs.

Reply:
I would have said Nordic Semiconductor SoCs. But then, nRF9160 is not an SoC, it's a SiP. So probably Nordic Semiconductor's hardware platforms then.
Are we ok with that?

Reply:
There must be an already established way to refer to this?

`Nordic Semiconductor devices`?

Reply:
Yes, that's the second way of writing it. I only found these two that we have mentioned now. So we go for your version then :)